resources/nl/cemu.po

```diff
@@ -551,116 +551,175 @@ msgstr ""
 "Beheert de uitvoer beeldverhouding wanneer het niet gelijk is aan de "
 "beeldverhouding van het spel."
 
-#: GeneralSettings2.cpp:235
-msgid "Overlay"
-msgstr "Overlay"
-
-#: GeneralSettings2.cpp:242
-msgid "Position"
-msgstr "Positie"
-
-#: GeneralSettings2.cpp:244 GeneralSettings2.cpp:698 GeneralSettings2.cpp:699
+#: GeneralSettings2.cpp:234 GeneralSettings2.cpp:765 GeneralSettings2.cpp:766
 msgid "Disabled"
 msgstr "Uitgeschakeld"
 
-#: GeneralSettings2.cpp:244
+#: GeneralSettings2.cpp:234
 msgid "Top left"
 msgstr "Links boven"
 
-#: GeneralSettings2.cpp:244
+#: GeneralSettings2.cpp:234
 msgid "Top center"
 msgstr "Midden boven"
 
-#: GeneralSettings2.cpp:244
+#: GeneralSettings2.cpp:234
 msgid "Top right"
 msgstr "Rechts boven"
 
-#: GeneralSettings2.cpp:244
+#: GeneralSettings2.cpp:234
 msgid "Bottom left"
 msgstr "Links onder"
 
-#: GeneralSettings2.cpp:244
+#: GeneralSettings2.cpp:234
 msgid "Bottom center"
 msgstr "Midden onder"
 
-#: GeneralSettings2.cpp:244
+#: GeneralSettings2.cpp:234
 msgid "Bottom right"
 msgstr "Rechts onder"
 
+#: GeneralSettings2.cpp:237
+msgid "Overlay"
+msgstr "Overlay"
+
+#: GeneralSettings2.cpp:244 GeneralSettings2.cpp:301
+msgid "Position"
+msgstr "Positie"
+
 #: GeneralSettings2.cpp:247
 msgid "Controls the overlay which displays technical information while playing"
 msgstr ""
 "Beheert de overlay die technische informatie weergeeft tijdens het spelen"
 
-#: GeneralSettings2.cpp:252
+#: GeneralSettings2.cpp:252 GeneralSettings2.cpp:310
 msgid "Text Color"
 msgstr "Tekst Kleur"
 
 #: GeneralSettings2.cpp:254
 msgid "Sets the text color of the overlay"
 msgstr "Veranderd de tekst kleur van de overlay"
 
-#: GeneralSettings2.cpp:264
+#: GeneralSettings2.cpp:259 GeneralSettings2.cpp:315
+msgid "Scale"
+msgstr ""
+
+#: GeneralSettings2.cpp:261
+#, fuzzy
+#| msgid "Sets the text color of the overlay"
+msgid "Sets the scale of the overlay text"
+msgstr "Veranderd de tekst kleur van de overlay"
```

Comment:
This hasn't been actually translated, it's the same string as "Sets the text color of the overlay"